frontend/packages/ux-editor/src/containers/DesignView/FormTree/FormItem/FormItem.tsx

```diff
@@ -46,7 +50,9 @@ export const FormItem = ({ layout, id, duplicateComponents }: FormItemProps) =>
       labelWrapper={labelWrapper}
       nodeId={id}
     >
-      {renderItemList(layout, duplicateComponents, id)}
+      {shouldDisplayAddButton
+        ? renderItemListWithAddItemButton(layout, duplicateComponents, id)
+        : renderItemList(layout, duplicateComponents, id)}
```

Comment:
An alternative approach 🤷‍♂️ 
```suggestion
{renderItemList(layout, duplicateComponents, id)}
{shouldDisplayAddButton && (<AddItem containerId={parentId} layout={layout} />)}
```

Reply:
I actually did this to begin with. But the `StudioDragAndDropTree.Item` seems to be very sensitive to the number/levels of divs within it. Doing it this way broke the Playwright tests, while using a specialized render function solved the issue.

Reply:
Ok 🙂 Thanks for the explanation 👍